configuring.tex

```diff
@@ -97,6 +97,25 @@ \section{Preparing for First Use: Formatting SD Cards}
 Record (``Boot Sector'') by instead typing ``FIX MBR,'' should the
 need arise.
 
+
+As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
+
+This is how it works: When powering on, MEGA65 will search for- and boot these files:
+o FREEZER.M65
```

Comment:
Lists need to look like this:

\begin{itemize}
\item {\tt FREEZER.M65}
\item {\tt AUDIOMIX.M65}
...
\end{itemize}

The {\tt ...} wrapper selects the C64 screen font which we use when mentioning filenames.

Reply:
Done